handler/area_types.go

```diff
@@ -0,0 +1,54 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-cantabular-dimension-api/contract"
+	"github.com/ONSdigital/dp-cantabular-dimension-api/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	ParamCantabularBlob = "cantabular_blob"
+)
+
+// AreaTypes handles requests to /area-types
+type AreaTypes struct {
+	respond responder
+	ctblr   cantabularClient
+}
+
+// NewAreaTypes returns a new AreaTypes handler
+func NewAreaTypes(r responder, c cantabularClient) *AreaTypes {
+	return &AreaTypes{
+		respond: r,
+		ctblr:   c,
+	}
+}
+
+// Get is the handler for GET /area-types
+func (at *AreaTypes) Get(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	blob := r.URL.Query().Get(ParamCantabularBlob)
+
+	res, err := at.ctblr.GetGeographyDimensions(ctx, blob)
+	if err != nil {
+		at.respond.Error(ctx, w, errors.Wrap(err, "failed to get area-types from cantabular"))
+		return
+	}
+
+	resp := contract.GetAreaTypesResponse{
+		AreaTypes: []model.AreaType{},
```

Comment:
Do you have to initialise this here? You might be able to get away with just `var resp contract.GetAreaTypesResponse`

Reply:
done